internal/build/phase_test.go

```diff
@@ -313,6 +313,20 @@ func testPhase(t *testing.T, when spec.G, it spec.S) {
 					h.AssertContains(t, outBuf.String(), `error connecting to internet:`)
 				})
 			})
+
+			when("#WithPlatformVolumes", func() {
```

Comment:
Is this test incomplete, or am I missing something? :)

Reply:
Whoops! This was a mistake, the tests for the platform prefix should all live in `build_test.go`